exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/Height.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2018 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.exonum.binding.qaservice;
+
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Objects;
+
+final class Height {
```

Comment:
If we are making a separate class for `height`, maybe we should make it `ulong` rather than `long`? It's `ulong` in Core. Or do you think that's excessive?

Reply:
I believe long is enough here.